app/models/user.rb

```diff
@@ -58,6 +58,8 @@ class User < ApplicationRecord
          :lockable, :confirmable, :trackable
 
   belongs_to :role, optional: true
+  has_many :ratings, dependent: :destroy
```

Comment:
Do movies have ratings?
Or just users have it?

Reply:
i think movie have ratings, and all ratings can created only by current_user, it's no need to create user associasion

Reply:
one rating has one movie and one user, but each movie could have many ratings and each user too